src/common/components/InputField/Input/Input.tsx

```diff
@@ -0,0 +1,44 @@
+import { ChangeEvent } from "react";
+import classNames from "classnames/bind";
+import styles from "./Input.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface InputProps {
+  type: "text" | "email" | "password" | "dropdown" | "search";
+  size?: "sm" | "md" | "full";
+  placeholder?: string;
+  name: string;
+  color?: "white" | "gray";
+  disabled?: true | false;
+  value?: string | number;
+  onChange?: (e: ChangeEvent) => void;
+}
+
+export default function Input({
+  type = "text",
+  size = "md",
+  placeholder = "",
+  name,
+  color = "white",
+  disabled = false,
+  value,
+  onChange,
+  ...rest
+}: InputProps) {
+  const className: string = cn("default", size, color);
```

Comment:
여기서는 자동 추론에 맡기면 어떨까 싶네요~
```suggestion
  const className = cn("default", size, color);
```

Reply:
현재 className이 복잡하지 않기 때문에 input 태그에 바로 넣어줘도 좋을 것 같습니다~